### DISCUSSION TITLE
Great Idea for a Addon

### DISCUSSION THREAD

**Discussion by gswashburn** [FlamxGames/godot-ai-assistant-hub#6] [General]
I run an Ollama Server in a Proxmox VM locally on my network, instead of locally on my machine, and was in process of designing an addon to use it in the Godot Editor. I have had some success with an addon in the past (Snippets Editor Plugin). If you consider everyone copying it, making one change then resubmitting it a success. My addon allowed you to have a folder of code snippets available to cut & paste within your project. I was going to start with that and add in an AI part. I like your design, although it is a bit complicated to initially setup for someone just starting with Godot. I had no issues getting it running in my environment using the remote Ollama instance I have running, but I have a lot of experience doing this type of thing. Your videos were a great help to get the example assistants and quick prompts working using the models I have running. I am interested to see where you will take this and what kind of traction you get with people using it. Keep up the good work.

**Comment by thedarknees45** [comment]
> I run an Ollama Server in a Proxmox VM locally on my network, instead of locally on my machine, and was in process of designing an addon to use it in the Godot Editor. I have had some success with an addon in the past (Snippets Editor Plugin). If you consider everyone copying it, making one change then resubmitting it a success. My addon allowed you to have a folder of code snippets available to cut & paste within your project. I was going to start with that and add in an AI part. I like your design, although it is a bit complicated to initially setup for someone just starting with Godot. I had no issues getting it running in my environment using the remote Ollama instance I have running, but I have a lot of experience doing this type of thing. Your videos were a great help to get the example assistants and quick prompts working using the models I have running. I am interested to see where you will take this and what kind of traction you get with people using it. Keep up the good work.

do you think you can help ?
